src/ape/api/providers.py

```diff
@@ -123,6 +143,11 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
     @property
     def confs(self) -> int:
+        """
+        NOT SURE WHAT THIS DOES
```

Comment:
Agree, not immediately obvious from the name!

Perhaps we can change this property to `confirmations` so it's more obvious

p.s. This is the number of blocks deep in the chain (starting from the "latest" head position of the chain and counting backwards) that this particular transaction is located. The higher this number, the less likely it is to be reorganized out of that block and into a block at a different height or orphaned entirely from the chain (see "Double Spending attack"). There is a practical consideration for what is considered an "acceptable" number of confirmations to wait for, which has to do with the relative value of the transaction (e.g. how much money is transferred or opportunity to profit is available) vs. the absolute rewards earned by the miners in that chain (block reward + uncle reward + transaction fees + mev income). For example, if you sent a low-value cryptokitty to someone else, the value of the transaction is very small and there is no other opportunity to profit, so you can wait a minimum number of confirmations (typically 6, because there is some instability in block production below that number and blocks can change). However, if you sent a Uniswap trade for $10m that causes a 2% price swing, that transaction has a high defined value ($20k opportunity) and is likely to see competition for MEV extraction, so you might want to wait 30 confirmations or more before assuming it is confirmed.

Reply:
I am not opposed at all to the rename, but this class probably just should not be documented and kept protected.
I don't want to have to worry about making breaking changes to a progress bar.

Reply:
Thank you for the clarification @fubuloubu 
ill rm doc string and wait for a refactor so i dont have to docstring this method.